nautobot/docs/development/core/templates.md

```diff
@@ -32,8 +35,12 @@ The most customizable template is `generic/object_detail.html`, as object detail
     * `nav_tabs`: are the navigation tabs. If overloaded, custom tabs can be
       rendered instead of the default.
         * `extra_nav_tabs`: this block allows to add new tabs without having to
-          override the default ones.
-* `content`: is the entire content of the page below the `header`.
+          override the default ones. Note that since v2.4.0 you can also define
+          `extra_tabs` in your view's [`object_detail_content`](ui-component-framework.md)
+          rather than overriding and extending the template.
+* `content`: is the entire content of the page below the `header`. Note that since v2.4.0,
+  if your view defines [`object_detail_content`](ui-component-framework.md), that content will
```

Comment:
Should we redirect them to the relevant section of the documentation? e.g. ObjectDetailContent Definition